lib/factree/dsl.rb

```diff
@@ -0,0 +1,40 @@
+# Readable shortcuts to common functions.
+module Factree::DSL
+  # Creates a {Conclusion} to return from a decision proc.
+  #
+  # The conclusion has an associated value (can be anything.)
+  #
+  # @param [Object] value Any value, including nil
+  # @return [Conclusion]
+  def conclusion(value=nil)
```

Comment:
👍  I'm trying to understand how an empty or false result happens. This sounds perfect since nil is never returned to the user.